model/src/guild/current_user_guild.rs

```diff
@@ -0,0 +1,60 @@
+use crate::{guild::Permissions, id::GuildId};
+use serde::{Deserialize, Serialize};
+
+/// A representation of a guild received when querying the http api for the current user guilds
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+pub struct CurrentUserGuild {
+    /// The ID of the guild
```

Comment:
Don't prefix with "the", otherwise most of them will have "the". Just "ID of the guild."